func/omp/stack_debug.cpp

```diff
@@ -1,58 +0,0 @@
-#include <faasm/faasm.h>
-#include <faasmp/faasmp.h>
-#include <omp.h>
-#include <stdio.h>
-
-/**
- * This function checks that OMP threads each have their own private stack.
- * The inner loop will always yield the same wasm offsets to refer to the
- * stack variables (as the code is exactly the same), but the runtime must
- * map these onto _separate_ stacks behind the scenes, rather than map them
- * all onto the same memory locations.
```

Comment:
This relies on the `__faasm_debug_copy` operation which I've removed to simplify things